modules/core/src/core/transform-shader-utils.js

```diff
@@ -1,17 +1,18 @@
 import assert from 'assert';
 
+const REGEX_DECLARATIONS = /^(#version[^\n]*\n)?/; // Beginning of file
```

Comment:
Don't we already have these in the shadertools inject code?

Can we avoid defining it twice?

Can some of this code go into shadertools